### PR TITLE
UCT/IB: Fixes and minor cleanups 

### PR DESCRIPTION
**Pull request by brminich**
- UCT/IB: Minor cleanups
- UCT/IB/MLX5: Fix tm_op in rc_hdr filling

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5913/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5918/ for details (Mellanox internal link).

File: src/ucs/sys/compiler_def.h [hunk 1]
```diff
@@ -126,6 +126,12 @@
         ucs_container_of(_ptr, _type, super); \
     })
 
+/**
+ * @return Size of _member in _type. _type is a structure type.
+ */
+#define ucs_sizeof(_type, _member) \
```
Comment (yosefe) [inline src/ucs/sys/compiler_def.h]: maybe ucs_field_sizeof ?

Comment (brminich) [reply src/ucs/sys/compiler_def.h]: ok, thought about `ucs_member_sizeof()` name :).  Let's go with `ucs_filed_sizeof`

Comment (yosefe) [reply src/ucs/sys/compiler_def.h]: either one is good

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5919/ for details (Mellanox internal link).

**Comment by brminich** [comment]
@yosefe, @hoopoepg, comment applied

**Pull request merged**